NEWS.md

```diff
@@ -31,6 +31,8 @@
   - `derive_var_ontrtfl(span_period)` 
   - `derive_var_shift(na_val)`
   - `derive_vars_aage(unit)`
+  
+- The function dplyr::transmute() is superseded in favor of dplyr::mutate(.keep = "none"). Consequently, all the admiral functions that utilized the former have been updated accordingly. (#2274)
```

Comment:
I would not consider it as a breaking change because it does not affect user code. I would move the item to the "Various" section.

Please use backticks for the function names, e.g., `dplyr::transmute()`.

Reply:
Moved it to 'Various' section.